src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,124 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.dukeexception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Storage object that handles the loading and unloading
+ * of tasks from/to the hard disk.
+ */
+public class Storage {
+    /** The file used to write/load data */
+    private File list;
```

Comment:
perhaps the File variable could be named in a different way to make it more clear its a file object